calibre-plugin/plugin-defaults.ini

```diff
@@ -2128,6 +2128,17 @@ cover_exclusion_regexp:/css/bir.png
 ## for those not expecting it.
 #append_datepublished_to_storyurl:false
 
+[touchfluffytail.org]
+use_basic_cache:true
+
+add_to_extra_valid_entries: averrating, reviews, comments, views
```

Comment:
When using `add_to_X` settings, the values need to start with `,`.

See the [add_to_keyword](https://github.com/JimmXinu/FanFicFare/wiki/INI-File#add_to_keyword) wiki entry, scroll down to where it says "The 'gotcha' about `add_to_` is..."